modules/js/src/core_bindings.cpp

```diff
@@ -520,6 +528,10 @@ EMSCRIPTEN_BINDINGS(binding_utils)
 
     emscripten::function("meanShift", select_overload<emscripten::val(const cv::Mat&, Rect&, TermCriteria)>(&binding_utils::meanShiftWrapper));
 
+    emscripten::function("cvtColor", select_overload<void(const cv::Mat&, cv::Mat&, int, int)>(&binding_utils::cvtColor_wrapper));
```

Comment:
I didn't see where those functions are used. Did I miss?

Reply:
It is useless because the image will be grayscale in C++ code. Should I delete this?

Reply:
Yes. Please do that.